addons/message_try_firefox_extension/downloadFirefox.js

```diff
@@ -0,0 +1,3 @@
+((api) => {
+  api.urlOpener.openUrlLabel('downloadFirefox');
```

Comment:
I don't like that to change the URL we have to change the client, which kind of makes the addon system unnecessarily brittle.

Reply:
I was just following convention here, and I don't love it either.

That said, nearly all addon changes take client work (for non-English languages), as we need to release new client versions to get updated translations - a brand new addon without a client update wouldn't be translated.

Since it's a nit, I'm going to leave this for now. If we want to change the convention, let's do it for all addons at once.